com.unity.mobile.notifications/Documentation~/Unified.md

```diff
@@ -115,14 +115,18 @@ To update an existing notification, schedule a notification with the same ID. Th
 If you want to check whether your application is launched by tapping a notification, you can query the last responded notification using the following code example:
 
 ```c#
-var notification = NotificationCenter.LastRespondedNotification;
-if (notification.HasValue)
+IEnumerator Start()
 {
-    var lastNotification = notification.Value;
+    var query = NotificationCenter.QueryLastRespondedNotification();
+    yield return query;
+    if (query.State == QueryLastRespondedNotificationState.HaveRespondedNotification)
+    {
+        var lastNotification = query.Notification;
+    }
 }
 ```
 
-This also works if you send notification to the background and tap the notification to bring it back to foreground. You can query the properties of returned notification.
+This also works if you send notification to the background and tap the notification to bring it back to foreground. You can query the properties of returned notification. The operation can be completed right away, but it is not guaranteed. Operation can be returned from coroutine to wait until it is complete, in case it was complete right away, it will continue execution on next iteration.
```

Comment:
```suggestion
This also works if you send the application to the background and tap the notification to bring it back to the foreground. You can query the properties of returned notification. Using QueryLastRespondedNotification API, you can track your query’s status and the notification details within the operation that the API returns as an object. You might receive the notification details immediately in the operation but it is not guaranteed. You can use the operation in a coroutine to wait until the query is complete. In case it completes immediately, the coroutine continues its execution on the next iteration.
```